vizro-ai/docs/pages/user-guides/customize-vizro-ai.md

```diff
@@ -62,6 +64,8 @@ vizro_ai = VizroAI(model="<chosen model>")
     | Base API URL      | `MISTRAL_BASE_URL`                        |
 
     To use Mistral with Vizro-AI, you can either use their API, which comes with [an associated cost](https://mistral.ai/technology/#pricing), or you could use their models for free under the Apache 2.0 license. In that case you need to setup the model API yourself. You can check [all available Mistral models including pricing on their website](https://docs.mistral.ai/getting-started/models/models_overview). This will also explain which version the below string acronyms currently point to.
+    Before usage, please review Mistral guidelines on risk mitigation to understand potential model limitations and best practices.
+    [See the Mistral site for more details on responsible usage](https://help.mistral.ai/en/collections/280418-legal-security-and-data-privacy/).
```

Comment:
Not sure this link applies, do you think this one might be better? https://help.mistral.ai/en/collections/272960-le-chat